internal/control-panel/registry.lua

```diff
@@ -77,6 +77,9 @@ COMMANDS_BY_IDX = {
         params={'--time', '1', '--timeUnits', 'days', '--command', '[', 'fix/stuck-instruments', ']'}},
     {command='fix/stuck-worship', group='bugfix', mode='repeat', default=true,
         params={'--time', '1', '--timeUnits', 'days', '--command', '[', 'fix/stuck-worship', ']'}},
+    {command='fix/zero-noexert-exhaustion', group='bugfix', mode='repeat', default=false,
+        desc='Prevent NOEXERT units (notably Necromancers, Vampires, and Intelligent Undead) from becoming Tired, and fix NOEXERT units which are Tired.',
+        params={'--time', '350', '--timeUnits', 'ticks', '--command', '[', 'fix/zero-noexert-exhaustion', ']'}},
```

Comment:
we have up to 450 ticks to run, so 350 might be a bit too much buffer. How about 439? It's closer to the limit and is prime, so there won't be resonance cycles with other repeat tools.

Reply:
I've seen Exhaustion reach as high as 1900 with the script on, so I don't think there's much headroom.